pkg/scheduler/api/devices/config/config.go

```diff
@@ -61,28 +62,33 @@ func GetConfig() *Config {
 	return configs
 }
 
-func loadConfigFromCM(kubeClient kubernetes.Interface, cmName string) (*Config, error) {
-	cm, err := kubeClient.CoreV1().ConfigMaps("kube-system").Get(context.Background(), cmName, metav1.GetOptions{})
+func loadConfigFromCM(kubeClient kubernetes.Interface, cmName, cmNamespace string) (*Config, error) {
```

Comment:
The configMap just load once, so if the scheduler faield to get configMap, user need to restart the scheuler?
If that's true, please add a trouble shooting in the user guide to let users be aware of that.

Reply:
If the configmap not exist or error, then the node will use hami-core instead.
Added the note.